### PR TITLE
E2e/project edit

### PR DESCRIPTION
**Pull request by Crevil**
# First batch of e2e tests

End 2 end testing of Project Edit and all tabs there.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,6 +5,7 @@
 *.suo
 *.user
 *.sln.docstates
+.vs/
```
Comment (snebjorn) [inline .gitignore]: Already ignored

File: Presentation.Web/App_Start/BundleConfig.cs [hunk 2]
```diff
@@ -26,8 +26,8 @@ public static void RegisterBundles(BundleCollection bundles)
                 "~/Scripts/angular-ui/ui-bootstrap-tpls.js",
                 "~/Scripts/ui-select2.js",
                 "~/Scripts/notify/*.js",
-                "~/Scripts/loading-bar.js",
-                "~/Scripts/angular-ui-util/ui-utils.js"));
+                "~/Scripts/angular-ui-util/ui-utils.js",
+                "~/Scripts/loading-bar.js"));
```
Comment (snebjorn) [inline Presentation.Web/App_Start/BundleConfig.cs]: What's the reason for this change?

File: Presentation.Web/Content/kitos.css [hunk 3]
```diff
@@ -350,7 +355,7 @@ div.task-usage-row {
         padding: 5px;
         margin: 5px;
     }
-
+
```
Comment (snebjorn) [inline Presentation.Web/Content/kitos.css]: whitespace

File: Presentation.Web/Partials/shared/org-tree-item.html [hunk 4]
```diff
@@ -1,7 +1,11 @@
 ﻿<div class="org-node">
     <span class="org-node-info">
         <label>
-            <input type="checkbox" value="{{ orgUnit.id }}" data-ng-model="orgUnit.selected" data-ng-change="save(orgUnit)" data-ng-disabled="!hasWriteAccess" />
+            <input type="checkbox"
+                   value="{{ orgUnit.id }}"
```
Comment (snebjorn) [inline Presentation.Web/Partials/shared/org-tree-item.html]: Missing controller as prefix

Comment (Crevil) [inline Presentation.Web/Partials/shared/org-tree-item.html]: This is right.
